platform/android/app/src/main/kotlin/coop/polypoly/polypod/PolyStyle.kt

```diff
@@ -0,0 +1,91 @@
+package coop.polypoly.polypod
+
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+data class PolyStyle(
+    val spacing: PolySpacing = PolySpacing(),
+    val radius: PolyRadius = PolyRadius(),
+    val font: PolyFont = PolyFont(),
+    val border: PolyBorder = PolyBorder()
+)
+
+data class PolySpacing(
```

Comment:
We should have a single source of truth for this and the Swift version. Don't know if shared code would do, but maintaining this at the same time as https://github.com/polypoly-eu/polyPod/blob/6e18e8ffe93f9e291bfdb93b75c17d038df7994e/platform/ios/PolyPodApp/PolyPod/PolyStyle.swift is going to be complicated.